consensus/babe/types.go

```diff
@@ -23,9 +23,9 @@ import (
 	"github.com/ChainSafe/gossamer/crypto/sr25519"
 )
 
-// BabeConfiguration contains the starting data needed for Babe
+// Configuration contains the starting data needed for Babe
 // see: https://github.com/paritytech/substrate/blob/426c26b8bddfcdbaf8d29f45b128e0864b57de1c/core/consensus/babe/primitives/src/lib.rs#L132
-type BabeConfiguration struct {
+type Configuration struct {
```

Comment:
Maybe `Config` to be consistent with the format elsewhere?

Reply:
Well, IDK, the naming is not of my creation. Maybe @noot could en-light on this one ?

Reply:
sure, `Config` is fine

Reply:
oks